src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -219,7 +219,14 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
                     {
                         // It's a number, formatted as date/time. Let's convert it to a date/time value first
                         var newDateTime = Library.NumberToDateTime(runner, context, IRContext.NotInSource(FormulaType.DateTime), new NumberValue[] { num });
-                        resultString = ExpandDateTimeExcelFormatSpecifiers(formatString, "g", newDateTime.Value, culture, runner.CancellationToken);
+                        try
```

Comment:
>try [](http://example.com/codeflow?start=24&length=3)

Nice touch adding this exception handler since it is possible to get an invalid format.
Would be a good idea (or worth the effort) to move this try/catch into ExpandDateTimeExcelFormatSpecifiers?

Reply:
Done